scalableminds/webknossos
# Fix generation of frontend documentation

which broke after the typescript conversion. Unfortunately, documentation.js doesn't seem to throw an error if the supplied file path cannot be found and I didn't find an option to enable that.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open the frontend API documentation or go to `/assets/docs/frontend-api/index.html`. The list of documented methods should not be empty.

------
(Please delete unneeded items, merge only when none are left open)
